packages/framework/src/Resources/scripts/common/validation/customizeBundle.js

```diff
@@ -441,14 +442,26 @@
 
     Shopsys.validation.showFormErrorsWindow = function (container) {
         var $formattedFormErrors = Shopsys.validation.getFormattedFormErrors(container);
+        var $window = $('#js-window');
+
+        if ($window.length === 0) {
+            Shopsys.window({
+                content:
+                    '<div class="text-left">'
```

Comment:
I would refactor this part of code, because it is identical as code few lines bellow and it violation with DRY principle